BatchExamples/DFIRBatch.reb

```diff
@@ -2693,6 +2693,181 @@ Keys:
         Recursive: false
         Comment: "Displays the location of the SQLite database associated with 4K Video Downloader"
 
+# Third Party Applications -> AnyDesk - https://anydesk.com
+
+    -
+        Description: AnyDesk
+        HiveType: SYSTEM
+        Category: Third Party Applications
+        KeyPath: CurrentControlSet\Services\AnyDesk
+        Recursive: true
+        Comment: "Displays artifacts relating to AnyDesk"
+# https://jsac.jpcert.or.jp/archive/2023/pdf/JSAC2023_1_1_yamashige-nakatani-tanaka_en.pdf
+
+# Third Party Applications -> Atera - https://www.atera.com
+
+    -
+        Description: Atera
+        HiveType: SYSTEM
+        Category: Third Party Applications
+        KeyPath: CurrentControlSet\Services\AteraAgent
+        Recursive: true
+        Comment: "Displays artifacts relating to Atera"
+# https://jsac.jpcert.or.jp/archive/2023/pdf/JSAC2023_1_1_yamashige-nakatani-tanaka_en.pdf
+
+# Third Party Applications -> ConnectWise (ScreenConnect) - https://screenconnect.connectwise.com/
+
+    -
+        Description: ConnectWise (ScreenConnect)
+        HiveType: SYSTEM
+        Category: Third Party Applications
+        KeyPath: CurrentControlSet\Services\ScreenConnect Client*
+        ValueName: DisplayName
+        Recursive: false
+        Comment: "Displays artifacts relating to ConnectWise (ScreenConnect)"
+# https://jsac.jpcert.or.jp/archive/2023/pdf/JSAC2023_1_1_yamashige-nakatani-tanaka_en.pdf
+
+# Third Party Applications -> LogMeIn - https://www.logmein.com
+
+    -
+        Description: LogMeIn
+        HiveType: SYSTEM
+        Category: Third Party Applications
+        KeyPath: CurrentControlSet\Services\LogMeIn
+        Recursive: true
+        Comment: "Displays artifacts relating to LogMeIn"
+# https://jsac.jpcert.or.jp/archive/2023/pdf/JSAC2023_1_1_yamashige-nakatani-tanaka_en.pdf
+
+# Third Party Applications -> RemoteUtilities - https://www.remoteutilities.com/
+
+    -
+        Description: RemoteUtilities
+        HiveType: SYSTEM
+        Category: Third Party Applications
+        KeyPath: CurrentControlSet\Services\RManService
+        Recursive: true
+        Comment: "Displays artifacts relating to RemoteUtilities"
+    -
+        Description: RemoteUtilities
+        HiveType: SYSTEM
+        Category: Third Party Applications
+        KeyPath: Usoris\Remote Utilities\RManService\Host\Parameters
+        Recursive: true
+        Comment: "Displays artifacts relating to Portable RemoteUtilities Configuration"
```

Comment:
This line and the one at 2764 should be flipped

Reply:
Fixed here, will PR - https://github.com/AndrewRathbun/RECmd/commit/73e7eaafc80eb0d3b28cd0d4bd6bb2ad7fbf9f25